quickwit-proto/proto/cluster.proto

```diff
@@ -42,6 +42,9 @@ message Member {
 
   /// If true, it means self.
   bool is_self = 3;
+
+  /// member reincarnation 
+  int64 generation = 4;
```

Comment:
uint64 maybe?

Reply:
We are using timestamps (that can be negative) for generation.